modules/calculators/Mileage.cpp

```diff
@@ -0,0 +1,25 @@
+/**
+ **  @file Mileage.cpp
+ **  @brief 走行距離を計算するクラス
+ **  @author molpui0726
+ **/
+
+#include "Mileage.h"
+
+// MotorController でモーターの角位置を int32_t 型で取得しているため、double ではなく int32_t
+// 型を使用する
```

Comment:
コメントの改行がなんか中途半端な場所になってます
```suggestion
// MotorController でモーターの角位置を int32_t 型で取得しているため、
// double ではなく int32_t 型を使用する
```